akka-docs/src/main/paradox/typed/cluster-sharding.md

```diff
@@ -87,20 +117,85 @@ is used but `tell` or any of the other @ref:[Interaction Patterns](interaction-p
 
 See @ref:[persistence](persistence.md) for more details.
 
```

Comment:
The "How it works" with link to that page is missing

Reply:
Good catch thanks.